src/components/Dropdown/index.jsx

```diff
@@ -1,11 +1,34 @@
+import { useEffect, useState } from "react"
 import DropdownButton from "../DropdownButton"
 
-function Dropdown() {
+function Dropdown() {  
+    const [unreadEmailCount, setUnreadEmailCount] = useState(0)
+
+    function getEmailCount() {
+        fetch("https://email-client-api.dev.io-academy.uk/emails")
+            .then(res => res.json())
+            .then(emailData => {
+
+                let unreadCount = 0
+
+                emailData.data.forEach(email => {
+                    if (email.read == 0){unreadCount ++}
+                })
+                setUnreadEmailCount(unreadCount) 
+            }
+    )}
+
+    useEffect(getEmailCount, [])
```

Comment:
It would be ideal if we could find a way to stop this from running every time the dropdown gets displayed - what you should do is move this function and useEffect up to the parent component (Nav I think) and then pass the emailCount down to this component as a prop.

That way the fetch request only gets sent once when the page first opens